allink_core/core/templates/partials/buttons/submit.html

```diff
@@ -0,0 +1,11 @@
+{% load i18n %}
+
+{% block control_group %}
+    <button type="submit"
+            {% if id %}id="{{ id }}"{% endif %}
```

Comment:
is there a reason we use a different naming of the context variables? in the link_ templates we use link_id and link_label

Reply:
I thought it would be nice to have generalized variables that could be used in different contexts.

We could also prefix everything with `button_`. But how would we handle the `icon` variable? It isn't prefixed in the link templates and changing it would possibly break older projects.

Reply:
do i understand you correctly? the plan would be to also change the context variables for all the link partials to generalized variables?

Reply:
That would be very consistent and flexible. But I can't estimate if that would lead to conflicts with other partials. Is that possible?

Reply:
the bigger issue is probably that we would need to change this in every project, we install the new core version. I don't think that would cause problems with other partials. (maybe sub-partials?) (But I like your approach o having simple/ non-prefixed context variables, it's a lot easier to read.)

Reply:
Then let's use the general variable names in the new button partial and keep the link partials as is.